src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
```

Comment:
You don't need extra line between parent and child elements.
Between siblings - yes, it's a good thing.